README.md

```diff
@@ -1,45 +1,20 @@
 # ds17800
 [![Build Status](https://travis-ci.com/Longi94/ds17800.svg?token=q2uGqSPNQRpjpMzytpn6&branch=master)](https://travis-ci.com/Longi94/ds17800)
 
-## Fault tolerance
-* Mirrored servers, no super nodes.
+## Build
+`./gradlew assemble`
 
-## Scalability
-* Middleware service for managing masters.
-* Entry point for the cliets. Get information about the masters.
-* Start up sleeping servers to scale performance.
+## Run
+Servers, dragons and players are run as Java jar files. Run one process for every server or entity with server/port as parameters.
 
-## Replication
-* Synchronized masters, replicated states.
-* Servers multicasts the command of their clients.
+#### Server
+`java -jar das-server/build/libs/das-server-1.0.0-SNAPSHOT.jar 10100`
 
-## Consistency
-* Synchronized masters, replicated states.
-* Race conditions are resolved by timestamp or id.
+#### Dragon
+`java -jar das-client/build/libs/das-client-1.0.0-SNAPSHOT.jar dragon 10100`
 
-### Etc.
-* Start with 3 servers.
-* Clients have the middleware IP.
-* Server handles action delays. On client.
-* TCP
+#### Player
+`java -jar das-client/build/libs/das-client-1.0.0-SNAPSHOT.jar player 10100`
 
-## Communication types
-
-### Middleware:
-* (Incoming)I am client, give me server to connect;
-* (Incoming) I am server, add me to server list;
-* (Outgoing) Server, are you still alive
-
-### Server:
-* (Incoming) I am client, here is my move
-* (Outgoing) Client, here is status of your move
-
-* (Outgoing) Server #i, here is my last moves, do you accept it
-* (Incoming) I am Server #i, your moves are accepted/rejected
-
-### Client:
-* (Outgoing) Hi middleware, I am new client, give me server to connect
-* (Outgoing) Server, it is my move
-  
-
-![arch](https://raw.githubusercontent.com/Longi94/ds17800/master/das.jpg)
+## Report
```

Comment:
I think they want us to add the pdf to the repo, we could also version control the latex source

Reply:
OK, let's download and `git add` the report when it's finished and we submit the project.